exercises/modules/readme.md

```diff
@@ -0,0 +1,76 @@
+# Modules
+
+Modules allow creating individual scopes and organise your code better. Read about [modules in cairo book](https://cairo-book.github.io/ch06-02-defining-modules-to-control-scope.html).
+
+Here's some code to show modules at play,
+
+## Declaring modules
```

Comment:
I'm actually wondering if we should give this explanation and not just redirect to the book. 
The more content we re-write, the more we have to maintain, and the more errors can slip in.

This part feels like a duplicate, it goes too much in-depth to be a high-order summary, and not enough to really understand everything

Reply:
Yeah, but Cairo book seemed a bit confusing to me.

Reply:
But it's there to refer to for more details...